pynicotine/headless/application.py

```diff
@@ -73,8 +74,30 @@ def on_confirm_quit(self, _only_on_active_uploads):
         cli.prompt(_("Do you really want to exit? %s") % responses, callback=self.on_confirm_quit_response)
 
     def on_invalid_password(self):
-        log.add(_("User %s already exists, and the password you entered is invalid. Please choose another username "
-                  "if this is your first time logging in."), config.sections["server"]["login"])
+
+        log.add(_("User %(user)s already exists, and the password you entered is invalid. Type %(command)s to "
+                  "log in with another username or password."), {
+            "user": config.sections["server"]["login"],
+            "command": "/connect"
+        })
+
+        config.sections["server"]["passw"] = ""
+
+    def on_setup_password_response(self, user_input):
+
+        config.sections["server"]["passw"] = user_input
+        config.write_configuration()
```

Comment:
Don't wipe the config if nothing was entered.

```
if not config.need_config():
    config.write_configuration()
```

Reply:
I agree with using the previous username when pressing enter, but I don't think old passwords should remain.